### DISCUSSION TITLE
Regarding use of Github

### DISCUSSION THREAD

**Discussion by open-source-god** [community/community#81598] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to effectively use github?

**Comment by mojoece** [comment, marked as answer]
Try to contribute more on open source projects, that way you will enhance your coding skills and also, you will learn about git and github.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @open-source-god! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
